inc/marketplace/view.class.php

```diff
@@ -658,7 +659,7 @@ static function getButtons(string $plugin_key = ""): string {
          } else if ($can_be_updated) {
```

Comment:
Maybe button title should be changed by `Download and install` 2 lines above.

Reply:
i try, as much as possible, to create new locales